tests/TestCase/I18n/I18nTest.php

```diff
@@ -514,6 +514,60 @@ public function testFallbackTranslator()
     }
 
     /**
+     * Test that the fallback translators can be disabled
+     *
+     * @return void
+     */
+    public function testFallbackTranslatorFallbackDisabled()
+    {
+        I18n::defaultFallbackDomain(false);
```

Comment:
You're right, probably better to have separat method.

But maybe positive wording e.g. `useTranslationFallback($enable = true)` even if it's enabled by default?